<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="mainSettingsGroupBox">
        <property name="title">
         <string>Ustawienia główne</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="conversionTypeLabel">
           <property name="text">
            <string>Typ konwersji</string>
           </property>
           <property name="buddy">
            <cstring>conversionType</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="conversionType"/>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="outputPathLayout">
           <item>
            <widget class="QLineEdit" name="outputPathLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="outputPathPushButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="outputPathLabel">
           <property name="text">
            <string>Ścieżka wyjściowa</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="sourcePathLayout">
           <item>
            <widget class="QLineEdit" name="sourcePathLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="sourcePathPushButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="sourcePathLabel">
           <property name="text">
            <string>Ścieżka źródłowa</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="sourceFileLayout">
           <item>
            <widget class="QLineEdit" name="sourceFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="sourceFilePushButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="sourceFileLabel">
           <property name="text">
            <string>Plik źródłowy</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <layout class="QHBoxLayout" name="listFileLayout">
           <item>
            <widget class="QLineEdit" name="listFileLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="listFilePushButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="listPathLabel">
           <property name="text">
            <string>Lista obrazków (.csv)</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QCheckBox" name="replaceExisting">
           <property name="text">
            <string>Zastąp już istniejące</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QCheckBox" name="checkSubdirs">
           <property name="text">
            <string>Sprawdzaj podkatalogi</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QComboBox" name="forcedFormatComboBox"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="forcedFormatLabel">
           <property name="text">
            <string>Wymuś format</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="conversionSettingsGroupBox">
        <property name="title">
         <string>Ustawienia kowersji</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="ratioLabel">
           <property name="text">
            <string>Proporcje</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="ratioLayout">
           <item>
            <widget class="QSpinBox" name="xRatio">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ratioSeparatorLabel">
             <property name="text">
              <string>:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="yRatio">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="marginLabel">
           <property name="text">
            <string>Marginesy (%)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="marginLayout">
           <item>
            <widget class="QSlider" name="marginSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="marginSpinBox">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="alphaToleranceLabel">
           <property name="text">
            <string>Tolerancja przezroczystości</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="alphaToleranceLayout">
           <item>
            <widget class="QSlider" name="alphaToleranceSlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="alphaToleranceSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="colorLabel">
           <property name="text">
            <string>Kolor tła</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <layout class="QHBoxLayout" name="colorLayout">
           <item>
            <widget class="QGraphicsView" name="colorView">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>8</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="colorButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Wybierz kolor</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="colorToleranceLayout">
           <item>
            <widget class="QSlider" name="colorToleranceSlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="colorToleranceSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="colorToleranceLabel">
           <property name="text">
            <string>Tolerancja koloru tła</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QCheckBox" name="clearColorCheckBox">
           <property name="text">
            <string>Szparuj zdjęcie</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="conversionStatusGroupBox">
        <property name="title">
         <string>Status kowersji</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QProgressBar" name="progressProgressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="actionsLayout">
           <item>
            <widget class="QPushButton" name="runButton">
             <property name="text">
              <string>Uruchom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="text">
              <string>Anuluj</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConvert">
    <property name="title">
     <string>Konwersja</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionCancel"/>
   </widget>
   <addaction name="menuConvert"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRun">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Uruchom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="text">
    <string>Anuluj</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
